script/_initrc

```diff
@@ -28,8 +28,12 @@
 #
 
 set -euo pipefail
-_HERE=$(dirname "$0")
-cd "${_HERE}/.."
+
+if [[ ${OTBR_VERBOSE:-} == 1 ]]; then
+    set -x
```

Comment:
Would it be useful to unconditionally set -x?

Reply:
I think this is only necessary when we want to debug the script. Otherwise, it just make it harder to find the real useful information.

Reply:
My primary concern is whether a new/casual user will know to set `OTBR_VERBOSE=1` when trying to debug something? There seems to be a lot of setup-related issues occurring with OTBR lately.

Reply:
Indeed, it should be worthwhile for setup to always have `set -e`.